packages/core/CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="2.5.1"></a>
+## [2.5.1](https://github.com/lerna/lerna/compare/1.0.0...2.5.1) (2017-11-18)
```

Comment:
This doesn't seem like correct output?

Reply:
I'll have a look at the broken changelog, don't know what happened there